examples/getting_started/user_config.yaml.example

```diff
@@ -2,21 +2,23 @@
 # Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
 #
 
-OPENBOT_ENV:
-    # Motor torque: 1200 gf.cm (gram force centimeter) == 0.1177 N.m
-    # Gear ratio: 50
-    # Max wheel torque: 5.88399 N.m
-    # https://www.conrad.de/de/p/joy-it-com-motor01-getriebemotor-gelb-schwarz-passend-fuer-einplatinen-computer-arduino-banana-pi-cubieboard-raspbe-1573543.html
-    
-    MOTOR_VELOCITY_CONSTANT: 65.89 # Velocity constant of the OpenBot motors
-    GEAR_RATIO: 50.0               # Gear ratio of the OpenBot motors
+EXAMPLES:
```

Comment:
Let's move this to a `default_config.yaml` in the `common` folder and pass it the path when calling `get_config(...)`. We went to all the trouble of defining the class in `common` so we wouldn't need to repeat ourselves in each example. It's weird to go to all of that trouble and still need to repeat ourselves in each `user_config.yaml.example` file.